owner_alice/create_layout.py

```diff
@@ -41,7 +45,11 @@ def main():
               ["CREATE", "demo-project.tar.gz"], ["DISALLOW", "*"],
           ],
           "pubkeys": [key_carl["keyid"]],
-          "expected_command": "tar --exclude '.git' -zcvf demo-project.tar.gz demo-project",
+          "expected_command": [
+              "tar",
+              "--exclude '.git'"
+              "-zcvf demo-project.tar.gz demo-project",
+          ],
```

Comment:
I get the following warnings when running the demo:
```
Run command '[u'tar', u'--exclude', u'.git', u'-zcvf', u'demo-project.tar.gz', u'demo-project']' differs from expected command '[u'tar', u"--exclude '.git'-zcvf demo-project.tar.gz demo-project"]'
```

This patch will get rid of the warnings:
```patch
diff --git a/owner_alice/create_layout.py b/owner_alice/create_layout.py
index 621d887..e4e33e1 100644
--- a/owner_alice/create_layout.py
+++ b/owner_alice/create_layout.py
@@ -47,8 +47,11 @@ def main():
           "pubkeys": [key_carl["keyid"]],
           "expected_command": [
               "tar",
-              "--exclude '.git'"
-              "-zcvf demo-project.tar.gz demo-project",
+              "--exclude",
+              ".git",
+              "-zcvf",
+              "demo-project.tar.gz",
+              "demo-project",
           ],
           "threshold": 1,
         }],
```